Nodejs/Product/ProjectWizard/ImportWizard.cs

```diff
@@ -45,6 +46,8 @@ public sealed class NewProjectFromExistingWizard : IWizard {
                     dte = new ServiceProvider(provider).GetService(typeof(DTE)) as DTE;
                 }
             }
+
+            bool addingNewProject = false;
```

Comment:
This is to record the ``AddingNewProject`` before it gets reset. The ``AddingNewProject`` will be set in the ``AfterExecute`` before the ``ImportWizardCommand`` gets raised, so I need to record the value.